e2e/edx_courses_config.template.json

```diff
@@ -28,6 +28,23 @@
             "entrance_exam_id": "",
             "self_paced": false
         },
+        "assets": {
+            "video": {
+                "body": {
+                    "contentType": "video/mp4",
+                    "filename": "video.mp4"
+                },
+                "display_name": null,
+                "content_type": null,
+                "url": null,
+                "date_added": null,
+                "id": null,
+                "portable_url": null,
+                "thumbnail": null,
+                "locked": null,
+                "external_url": null
```

Comment:
Are they default values ?

Reply:
Yes, Apropos, they are not required, however, it might be useful to know the available values that can be set up.